os-eap-launch/added/launch/access_log_valve.sh

```diff
@@ -36,9 +36,15 @@ function configure() {
 }
 
 function configure_access_log_valve() {
-    EAP6_VALVE="<valve name=\"accessLog\" module=\"org.jboss.openshift\" class-name=\"org.jboss.openshift.valves.StdoutAccessLogValve\">\n              \
-    <param param-name=\"pattern\" param-value=\"%h %l %u %t %{X-Forwarded-Host}i \&quot;%r\&quot; %s %b\" />\n        \
-    </valve>"
+
+    EAP6_VALVE="<valve name=\"accessLog\" module=\"org.jboss.as.web\" class-name=\"org.apache.catalina.valves.AccessLogValve\">\n \
+      <param param-name=\"pattern\" param-value=\"%h %l %u %t %{forwarded}i \&quot;%r\&quot; %s %b\" />\n \
+      <param param-name=\"directory\" param-value=\"/proc/self/fd\" />\n \
+      <param param-name=\"rotatable\" param-value=\"false\" />\n \
+      <param param-name=\"prefix\" param-value=\"1\" />\n \
```

Comment:
I believe this will lead to corruption in the messages at some point, depending on how they are buffered.